rfcs/readme.md

```diff
@@ -0,0 +1,66 @@
+# OpenUI Design System RFC
+
+The OpenUI Design System is meant to provide a global, accessible, and themable design system for the web platform by creating high-quality and accessible components that work across any web framework.
+
+The normal GitHub pull request workflow will be used to provide bug fixes and documentation improvements to existing components, however creating new components or significantly modifying existing ones by creating variations will need to have more thorough review/discussion.
+
+This RFC (request for comments) process will allow anyone to submit issues using the provided template to begin discussions and provide the ability for new components to make their way into the design system.
+
+## When to follow this process
+
+It is necessary to follow the RFC process when you are intend to create any significant changes to the OpenUI Design System. Examples of these types of changes are:
+
+* A change which provides significantly alters existing functionality, provides new functionality, or a new primative component.
+* The removal of an existing feature or functionality
+
+Another way to look at it is, if your suggested change will require a new semantic version, or breaking change to an existing feature, it will need to go through the RFC process.
```

Comment:
We should detail how we intend to do versioning, you reference here so it implies we'll leverage the common semver paradigm but it would be good to call this out in the Readme or elsewhere. Not a blocking change to the PR.